app/helpers/layout_helper.rb

```diff
@@ -21,7 +21,7 @@ def stylesheet(*args)
   end
 
   def javascript(*args)
-    content_for(:javascripts) { javascript_include_tag(*args) }
+    content_for(:javascripts) { javascript_include_tag(*args.push("data-turbolinks-track" => true)) }
   end
```

Comment:
@ohadlevy, fixed in commit to squash. 

Also, there is `stylesheet_link_tag` and `javascript_include_tag` in 
https://github.com/theforeman/foreman/blob/develop/app/views/puppetclasses/no_route.html.erb#L1-L2 but I didn't change these. In fact, searching the code, I don't see where this view is called.